utils/preprocess_utils.py

```diff
@@ -22,34 +22,38 @@ def __init__(self, path):
             Metadata column names to be kept in the retrieved dataframe (optional)
         """
         path = pathlib.Path(path)
-                     
+
         # If the file isn't found in the path above then raise an error.
         if not path.is_file():
             raise FileNotFoundError(f"File '{path}' does not exist")
-            
-        if 'parquet' in path.name:
+
+        if "parquet" in path.name:
             self.df = pd.read_parquet(path)
-            
-        elif 'csv' in path.name:
+
+        elif "csv" in path.name:
             self.df = pd.read_csv(path)
-            
-        elif 'tsv' in path.name:
-            self.df = pd.read_csv(path, delimiter='\t')
-            
+
+        elif "tsv" in path.name:
+            self.df = pd.read_csv(path, delimiter="\t")
+
         else:
-            raise ValueError("The file must be a parquet, csv, or tsv, with the applicable extension included in the filename.")
+            raise ValueError(
+                "The file must be a parquet, csv, or tsv, with the applicable extension included in the filename."
+            )
 
-        self.df = self.df.loc[:,self.df.columns != 'Unnamed: 0'] # Remove the unnamed column
+        self.df = self.df.loc[
+            :, self.df.columns != "Unnamed: 0"
+        ]  # Remove the unnamed column
```

Comment:
Where does the unnamed column come from? Consider documenting this here or elsewhere.

Reply:
Documented! This was just an extra step, just in case there are any unnecessary columns.